Automattic/wp-calypso
# Jetpack: clone settings

#### Changes proposed in this Pull Request

Adding the demo of Jetpack Clone Settings functionality.
p9dueE-4ST-p2

#### Testing instructions
1. Create two Jetpack sites, connect them. Make sure both sites have the same plan.
2. Go to Site A and change some settings (e.g. the ones in Jetpack -> Settings -> Security). Do not make the same changes on Site B.
3. Go to `jetpack-clone-settings/your-siteB-url.local` and confirm that you see the "Clone Settings" UI: <img width="755" alt="Screen Shot 2022-05-17 at 11 45 40" src="https://user-images.githubusercontent.com/1341249/168853876-b1865a54-dab8-4e90-af54-4343e12c7193.png">
4. Choose "Site A" in the dropdown, click "Clone Settings". Confirm that you see the "success" notice in the top right corner (disappears after a few seconds).
5. Go to "Site B", confirm the settings you changed on "Site A" were copied there.

#### Tasks

- [x] Build the demo UI
- [x] Make the UI call the endpoint
- [x] Display endpoint response (success/error) as Calypso notifications
- [x] Make the page available only for team members
